client/components/deposits-overview/index.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Card, CardHeader } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import { useAllDepositsOverviews } from 'wcpay/data';
+
+interface OverviewProps {
+	overview: AccountOverview.Overview | null;
+	account: AccountOverview.Account | null;
+	isLoading: boolean;
+}
+
+/**
+ * Renders a deposits overview
+ *
+ * @param {AccountOverview.Overview} props Deposits overview
```

Comment:
Suggestion: I think this JSDoc param type should match the type provided to the function declaration below: `const DepositsOverviewDetails: React.FunctionComponent< OverviewProps > = (`
```suggestion
 * @param {OverviewProps} props Deposits overview
```

Reply:
Fixed in [8459303](https://github.com/Automattic/woocommerce-payments/pull/5759/commits/8459303f34be76f2e563134f32dac3e09a8e3a7b)